resources/views/model.blade.php

```diff
@@ -12,12 +12,13 @@
     }
 @endphp
 
-
-@if($fillable->contains("created_by_admin_user_id")|| $fillable->contains("updated_by_admin_user_id"))
-    use Brackets\AdminAuth\Models\AdminUser;
-@endif
 use Illuminate\Database\Eloquent\Model;
-
+@if($fillable)@foreach($fillable as $f)
+@if($f == "created_by_admin_user_id")use Brackets\Craftable\Traits\CreatedByAdminUserTrait;
+@elseif($f == "updated_by_admin_user_id")use Brackets\Craftable\Traits\UpdatedByAdminUserTrait;
```

Comment:
Do we have this traits prepared?

Reply:
Yes, it is prepared in craftable package in publishable pull request